From c234621caf8d5203e9910a93af0f440eef48b0e0 Mon Sep 17 00:00:00 2001
From: Evan Wang <xswang@marvell.com>
Date: Wed, 18 Jan 2017 00:27:48 +0800
Subject: [PATCH 1403/2241] dts: phy: a3700: update udc dts node to connect to
 phy handler

- for udc, phy handler is utmi_usb32

Change-Id: Id7a6a60ff86fef33c34947567c791fb42d92f254
Signed-off-by: Evan Wang <xswang@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/35707
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Victor Gu <xigu@marvell.com>
---
 arch/arm64/boot/dts/marvell/armada-3720-db.dts | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/marvell/armada-3720-db.dts b/arch/arm64/boot/dts/marvell/armada-3720-db.dts
index d2b8bb0..4348409 100644
--- a/arch/arm64/boot/dts/marvell/armada-3720-db.dts
+++ b/arch/arm64/boot/dts/marvell/armada-3720-db.dts
@@ -141,6 +141,8 @@
 
 			udc@54100 {
 				status = "disabled";
+				phys = <&utmi_usb32>;
+				phy-names = "usb";
 			};
 
 			/* CON31 */
-- 
2.7.4

